src/observe-tag.js

```diff
@@ -13,9 +13,28 @@ module.exports = tagFunction => {
 	const tagReaction = observe(() => {
 		// if there is an existing tagResult, it is the last rendering, and so we want to re-render over it
 		let oldTag = tagResult
+		const removedElementWithFocusData = {
+			index: null,
+			selectionStart: null,
+			selectionEnd: null,
+			selectionDirection: null
+		}
 
 		// remove oldTag first so that we unobserve before we re-observe
 		if (oldTag) {
+			// if there was focus, we need to figure out what element has it
+			const children = oldTag.querySelectorAll('*')
```

Comment:
Is focus the only data that needs this kind of special casing? Should we look out for other similar meta data that needs to be kept between updates? (Idk if undo states or anything else is special in this regard)

Reply:
I don't know of other ones off-hand... if there are we can make issues for them. 
Focus might just be the most obvious. If another shows up, then we can look at making a more generic solution.